indexd/drs/blueprint.py

```diff
@@ -50,7 +50,6 @@ def get_signed_url(object_id, access_id):
 
 
 def indexd_to_drs(record, list_drs=False):
-    bearer_token = flask.request.headers.get("AUTHORIZATION")
```

Comment:
So we still have to accept the auth token but we're just going to ignore it? cc @Avantol13 if you're cool with it then I"m not picky about it either.

Reply:
Oh that's been removed .

Reply:
yeah, that's fine by me